tests/components/bsblan/conftest.py

```diff
@@ -58,6 +58,11 @@ def mock_bsblan() -> Generator[MagicMock]:
         bsblan.sensor.return_value = Sensor.from_json(
             load_fixture("sensor.json", DOMAIN)
         )
+        bsblan.hot_water_state.return_value = HotWaterState.from_json(
+            load_fixture("dhw_state.json", DOMAIN)
+        )
+        # mock get_temperature_unit property
+        type(bsblan).get_temperature_unit = property(lambda self: "°C")
```

Comment:
```suggestion
        bsblan.get_temperature_unit = "°C"
```
I think this would work as well